releng/org.eclipse.nebula.nebula-parent/pom.xml

```diff
@@ -33,7 +33,7 @@ Contributors:
 		<jacoco-version>0.8.9</jacoco-version>
 		<easymock-version>5.2.0</easymock-version>
 
-		<target-platform>https://download.eclipse.org/releases/latest</target-platform>
+		<target-platform>https://download.eclipse.org/eclipse/updates/4.32-I-builds/</target-platform>

```

Comment:
FYI, replacing the release train site with the Platform's I-builds site is going to reduce the available set of 3rd party bundles significantly, leading to resolution failures.

Reply:
Yes I noticed that as well. But since I assumed it will take some time until this is submitted anyway and it was temporary I was lazy to look up how the repos are set up.
But now I checked it and it should work. :)

Reply:
Now the build succeeded :)

Reply:
After 4.32 M1 release you could change it to `https://download.eclipse.org/releases/2024-06/`?

Reply:
Yes that would be possible too, but would then later have to be changed back, if the latest is generally desired.

Reply:
FYI, during the end of the last release cycle SimRel produced and now maintains this:

https://download.eclipse.org/releases/milestone/

as requested here:

https://github.com/eclipse-simrel/simrel.build/issues/286

This site has the advantage that it tracks milestones and release candidates so that any problems are noticed before the release rather than after the release.